### PR TITLE
adds column to the first frame from the clojure compile message

### PR DESCRIPTION
**Pull request by timothypratley**
Shows the column as well as line number when present for the first frame.
It is useful for code highlighting with libraries other than PrismJS (such as codemirror)
which allow setting a cursor position.

### CODE REVIEW

File: test/prone/prep_test.clj [hunk 1]
```diff
@@ -60,7 +60,7 @@
                                                :age 37
                                                :url (java.net.URL. "http://example.com")
                                                :body (ByteArrayInputStream. (.getBytes "Hello"))
-                                               :closed-stream (doto (io/input-stream "http://example.com") .close)
+                                               :closed-stream (doto (io/input-stream "http://google.com") .close)
```
Comment (magnars) [inline test/prone/prep_test.clj]: Not that it matters, but why this change? :-)

**Comment by magnars** [comment]
Looks good to me. Are you in fact using codemirror with Prone? If so, how? Is it better? :-)

**Comment by timothypratley** [comment]
Hi Magnar,

Well, right now I wouldn't say there is much difference... this is what it looks like:

prone-codemirror

Here is the CodeMirror component that displays it:
https://github.com/timothypratley/fatwheel/blob/master/src/fatwheel/client/view/code_excerpt.cljs

One idea that might make it useful is that if I allow users to change and then save the file right there in browser then I don't have to find the error in my editor for a small fix... but it is just an idea at the moment.

As to replacing example.com with google.com:
When I run the tests locally with `lein test` I get an exception:

      actual: java.io.FileNotFoundException: http://example.com

But it seems to pass on the travis build /shrug ... I've reverted it

Regards,
Timothy

**Comment by magnars** [comment]
That's a cool idea. :-) Thanks for the PR!

**Comment by magnars** [comment]
This and #50 released as `1.6.0`.

**Comment by timothypratley** [comment]
Great, thank you!
